### DISCUSSION TITLE
Develop Pinning strategy

### DISCUSSION THREAD

**Discussion by ChristianChiarulli** [LunarVim/LunarVim#1785] [Ideas]
We need to pin our core plugins to specific commits for reproducibility and stability.

- how often should they be updated? 
  - weekly? 
  - monthly? 
  - never? 
  - when we feel like it?

**Comment by abzcoding** [comment]
weeky

**Comment by ahmedkhalf** [comment]
I personally want it 8 days instead of one week.

EDIT:

Seriously speaking though, I think it should be pinned by release. The master branch should always be pinned, and the rolling should always be unpinned. Then once the rolling merges with the master, the new pin is updated to reflect the latest rolling.

**Comment by dhollinger** [comment]
What about specific versions of neovim. Unless you're using an appimage or pinning brew/apt/dnf packages that will get updated on a regular schedule. Is there a plan to bundle neovim with the pinned versions? Does pinning Neovim even need to be done?

**Comment by lillianspencer796** [comment]
BoxesGen offers premium **[wholesale packaging solutions](https://boxesgen.com/)**, combining innovation and affordability to enhance your brand's appeal. Our customizable options and eco-friendly materials ensure a tailored and sustainable packaging experience for your products.
